docs/release_notes/plugin_api/0.1.x.rst

```diff
@@ -0,0 +1,8 @@
+Plugin API 0.1.alpha Release Notes
```

Comment:
Actually we need to have a hyphen separating the alpha part:  http://semver.org/#spec-item-9